packages/encoding/src/readState.ts

```diff
@@ -0,0 +1,8 @@
+import { ByteReader } from './ByteReader'
+import { decodeState } from './decodeState'
+
+export function readState(reader: ByteReader) {
```

Comment:
Do we have some docs about this data layout that we could mention in the readme/comments?